contrib/builder/deb/generate.sh

```diff
@@ -73,6 +73,13 @@ for version in "${versions[@]}"; do
 		extraBuildTags+=' exclude_graphdriver_btrfs'
 	fi
 
+	# make backports the default for installing packages
+	if [ "$suite" = 'wheezy' ]; then
+		#   remove btrfs-tools so we can readd it and pull from backports
+		packages=( "${packages[@]/btrfs-tools}" )
+		packages+=('--no-install-recommends && apt-get install -y -t wheezy-backports btrfs-tools')
```

Comment:
Also, I think this means we need to update `project/PACKAGERS.md` too, since it says:
> - btrfs-progs version 3.16.1 or later (unless using an older version is absolutely necessary, in which case 3.8 is the minimum)